tests/vmi_multus_test.go

```diff
@@ -923,7 +923,6 @@ func checkMacAddress(vmi *v1.VirtualMachineInstance, interfaceName, macAddress s
 		&expect.BExp{R: "#"},
```

Comment:
You can also fix this to: `PromptExpression`

Reply:
Done